NEWS.md

```diff
@@ -2,33 +2,111 @@ Recent Changes in Lepton EDA
 ============================
 
 This file documents important user-visible changes in Lepton EDA.  For
-more information, please consult the `ChangeLog` file.
```

Comment:
Shouldn't we delete top-level `ChangeLog` file (or at least bring it up to date)?

Reply:
OK, I've deleted it.